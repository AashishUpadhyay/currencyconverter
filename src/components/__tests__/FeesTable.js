import React from 'react';
import FeesTable from '../FeesTable';
import renderer from 'react-test-renderer';

describe('The fees table comonent', () => {

    it("renders as expected", () => {
        const tree = renderer.create(<FeesTable
            originCurrency={'USD'}
            destinationCurrency={'EUR'}
            conversionRate={1.3}
            fee={10}
            total={101}
        />).toJSON();

        console.log(tree);

        expect(tree).toMatchSnapshot();
    });
});