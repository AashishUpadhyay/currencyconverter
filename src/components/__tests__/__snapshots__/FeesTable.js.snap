// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The fees table comonent renders as expected 1`] = `
<div>
  <table>
    <tbody>
      <tr>
        <td>
          Conversion Rate
        </td>
        <td>
          1 
          USD
           -&gt; 
          1.30
           
          EUR
        </td>
      </tr>
      <tr>
        <td>
          Fee
        </td>
        <td>
          10.00
           
          USD
        </td>
      </tr>
      <tr>
        <td
          className="total-label"
        >
          Total Cost
        </td>
        <td>
          101.00
           
          USD
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
